pkg/container/qemu_runner.go

```diff
@@ -118,8 +118,37 @@ func (bw *qemu) Run(ctx context.Context, cfg *Config, envOverride map[string]str
 
 func (bw *qemu) Debug(ctx context.Context, cfg *Config, envOverride map[string]string, args ...string) error {
 	clog.FromContext(ctx).Debugf("running debug command: %v", args)
-	clog.InfoContextf(ctx, "To enter this environment: ssh root@localhost -p %s",
-		strings.Split(cfg.SSHAddress, ":")[1])
+
+	// default to root user, unless a different user is specified
+	user := "root"
+	if cfg.RunAs != "" {
+		user = cfg.RunAs
+	}
+
+	log.Debug("qemu: ssh - get user ssh key pair")
+	pubKey, err := getUserSSHKey()
+	if err != nil {
+		log.Warn("qemu: could not get user ssh key pair, using ephemeral ones")
+	}
+	if pubKey != nil {
+		command := fmt.Sprintf("echo '%s' | tee -a /root/.ssh/authorized_keys /home/*/.ssh/authorized_keys", string(pubKey))
+		err := sendSSHCommand(ctx,
+			"root",
```

Comment:
Won't this break using our ephemeral key pair?

When debugging you can `exit 0` to continue the build, and I think this breaks that?

Reply:
It shouldn't, `tee -a` is appending them

Reply:
Confirm this works as intended